src/e2ee.rs

```diff
@@ -345,14 +306,15 @@ fn contains_report(mail: &ParsedMail<'_>) -> bool {
 ///
 /// If this succeeds you are also guaranteed that the
 /// [Config::ConfiguredAddr] is configured, this address is returned.
+// TODO, just remove this?
```

Comment:
If it is not used, let's remove before we forget about it.

Reply:
Yes, I absolutely plan to do this but wanted to do that in a separate PR.  It touches a number of unrelated places which also still need to be touched as the rest of the key.rs api is transitioned.  So my plan was to remove this once the rest is done if it wasn't automatically removed by then.

I've updated the comment to actually say this.